roboteq_driver/mbs/script.hex

```diff
@@ -1,26 +1,35 @@
-:10000000020801060202020106020901020127029A
-:100010000801026E270213020102012702130202E5
-:1000200002012702020103BC0227020301032C0183
-:100030002702480201023227024902010300042775
-:100040000236020103AC0D27022A020103DC052758
-:10005000022C020103D00727022D020102322702DF
-:100060002B020101270233020103204E2702340232
-:100070000103204E2702270201020127022E02015E
-:10008000021427022F020102282702300201020A6D
-:10009000270232020102642702380201012702010D
-:1000A0000202060202050201051D03CA001F021614
-:1000B0000202052A02011803BF001F03EA00210201
-:1000C00002051402020603A3002002030514020322
-:1000D0000602030502051803E3001F035201210174
-:1000E000020306021323039E0020240326663A021D
-:1000F00002052524013A010202052A2524013A02BB
-:10010000010202052A2524013A02020202052A25DB
-:1001100024013A02030202052A2524013A020402BC
-:1001200002052A2524013A020D02022A2524013A59
-:10013000020C0202052A2524013A02100202052AB5
-:100140002524013A02120202050201092A2524018E
-:100150000D22240326733A01052524013A021501D4
-:100160002A2524013A0214012A2524013A020D020B
-:10017000012A2524013A020D02032A2524013A020C
-:100180001202012A2524010D2200FFFFFFFFFFFFBD
+:1000000002170106020202010602010F02020201AA

```

Comment:
This hex file is unfortunately generated using the vendor's Windows-only tool; it's not possible to generate it at build-time. As a compromise, both the MBS "source" file and hex output are committed to the repo.